library.properties

```
@@ -0,0 +1,10 @@
+name=ArTICL
+version=0.0.1
```

```suggestion
version=1.0.0
```
It's been around for long enough that let's call this a first major version, at least. Properly recording version numbers is probably de rigueur now.

---

The version number with an associated commit tag is how the Arduino Library will track releases at least